crates/kubelet/src/node.rs

```diff
@@ -389,3 +383,66 @@ fn lease_spec_definition(node_name: &str) -> serde_json::Value {
         }
     )
 }
+
+/// Defines the labels that will be applied to this node
+///
+/// Default values and passed node-labels arguments are injected by config.
+fn node_labels_definition(arch: &str, config: &Config) -> HashMap<String, String> {
```

Comment:
Could we add a unit test for this method?

Reply:
I tried, but struggled as it isn't a public method, and none of the other foo_definition methods in this class are public, and I didn't want to A) break what appears to be an intentional design or B) start introducing DI which is how I would do it in other languages.  I'm new to rust.  I'd be open to some help or advice here if you have any.

Reply:
For unit tests, they are usually in the same file, e.g. https://doc.rust-lang.org/book/ch11-01-writing-tests.html. A real-world example: https://github.com/bytecodealliance/wasmtime/blob/7eea5d8d438bf0d65710410c0acc58e5b27018d7/crates/runtime/src/vmcontext.rs#L53-L62.

Reply:
Oh wow!  I had no idea you could just super:: out of the test mod like that.  Thx.  I'm giving it a whirl.

Reply:
was this resolved?

Reply:
yes.  Unit test has been added